test/ElasticSearchRepositoryFactoryTest.php

```diff
@@ -22,7 +22,7 @@ class ElasticSearchRepositoryFactoryTest extends TestCase
     /**
      * @test
      */
-    public function it_creates_an_elastic_search_repository()
+    public function itCreatesAnElasticSearchRepository(): void
```

Comment:
hmm, this is possible enforced by a newer version of php-cs-fixer. In `broadway/broadway` we prevented this by https://github.com/broadway/broadway/blob/master/.php_cs#L14-L16.

I like to keep the test methods names in snake case.

Reply:
Yep, I prefer snake case as well, just got surprised but the phpcs rules, I'll get them back